automated/linux/iperf/iperf.sh

```diff
@@ -65,6 +67,20 @@ fi
 
 # Run local iperf3 server as a daemon when testing localhost.
 if [ "${SERVER}" = "" ]; then
+    cmd="lava-echo-ipv4"
+    if which "${cmd}"; then
+        ipaddr=$(${cmd} "${ETH}" | tr -d '\0')
+        if [ -z "${ipaddr}" ]; then
+            echo "lava-test-raise ${ETH} not found";
```

Comment:
if lava-echo-ipv4 is defined this means we're running in lava so lava-test-raise should also be defined. In such case don't use 'echo' but call lava-test-raise.

Reply:
Oh yeah!